package.json

```diff
@@ -2,11 +2,16 @@
   "name": "redux-promise-actions",
   "version": "0.5.1",
   "description": "Flux Standard Action async utilities for Redux.",
-  "main": "index.js",
+  "main": "dist",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "babel src --presets env --out-dir dist",
```

Comment:
This builds all the tests too, which would then ship with the npm module. I prefer not to publish tests to npm.